src/vs/workbench/services/positronConnections/browser/positronConnectionsInstance.ts

```diff
@@ -53,54 +50,74 @@ class BaseConnectionsInstance extends Disposable {
 
 export class PositronConnectionsInstance extends BaseConnectionsInstance implements IPositronConnectionInstance {
 
-	readonly onToggleExpandEmitter: Emitter<void> = new Emitter<void>();
-	private readonly onToggleExpand: Event<void> = this.onToggleExpandEmitter.event;
+	private readonly onDidChangeEntriesEmitter = new Emitter<IPositronConnectionEntry[]>();
+	readonly onDidChangeEntries = this.onDidChangeEntriesEmitter.event;
 
-	readonly onDidChangeDataEmitter: Emitter<void>;
+	private readonly onDidChangeStatusEmitter = new Emitter<boolean>();
```

Comment:
```
private readonly onDidChangeStatusEmitter = this._register(new Emitter<boolean>());
```

Reply:
Resolved in fe6a0d009ddab8ff0dc5ac6b3cfd3145ec8cdd97